themes/default/content/blog/announcing-pulumi-up-2022/index.md

```diff
@@ -1,7 +1,7 @@
 ---
-title: "Announcing PulumiUP 2022: The Cloud Awakens!"
+title: "Announcing PulumiUP 2022: The Cloud Awakens on May 4th"
 date: 2022-03-31
-meta_desc: Today we announce the second annual PulumiUP virtual conference taking place on May 4, 2022. Join us!
+meta_desc: Attend PulumiUP 2022 with virtual workshops, sessions, and talks about cloud engineering, infrastructure as code, and modern cloud architectures with great speakers!
```

Comment:
The linter's failing because this is over 150 characters. Maybe this?

```suggestion
meta_desc: Attend PulumiUP 2022 with virtual workshops, sessions, and talks about cloud engineering, infrastructure as code, and modern cloud architectures!
```